webapp/components/wpt-flags.js

```diff
@@ -57,7 +57,6 @@ Object.defineProperty(wpt, 'ServerSideFeatures', {
       'paginationTokens',
       'pendingChecks',
       'runsByPRNumber',
-      'serviceWorker',
```

Comment:
Probably need to delete this in Datastore on staging and prod

Reply:
Yup, I'll do that!